### PR TITLE
Handcuff crap 

### PR DESCRIPTION
**Pull request by wonderinghost**
# Document the changes in your pull request

adds sprite to alien handcuffs
holoparasites can now break handcuffs/legcuffs again (seems unintentionally removed) 

reminder but a strength range of 2 to 4 (D to B rank) will break them out 

swarmer handcuffs now take 20 seconds to break out of from 45
finally adds a sprite to alien cuffs 
converts cuff deci to seconds on cuffs

# Why is this good for the game?
we've all been there when a swarmer launches us across the station. its annoying that we are launched into a area we cant get out of with egregious break out times on the cuffs

holoparasites i came across by accident and when asking about it no one seems to know what happened to it (tattax probably), re enabling this unless there was a reason it was stealth removed

not sure why i edited handcuffs, probably because of buckle cuffing 

# Testing
checked multiple times to make sure things updated 

# Spriting
alien cuff sprite
aliencuffs

# Wiki Documentation

if holoparasites didn't have this feature then add it
random fact but wirecutters can break basic restraints on other people, standard handcuffs or weirder will need jaws of life 

# Changelog

:cl:  

bugfix: fixes holoparasites ability to break cuffs
tweak: swarmer cuffs take 20 seconds instead of 45
tweak: converts cuff deci seconds to seconds
bugfix: added sprite to alien cuffs

/:cl:

### CODE REVIEW

**Review by Moltijoe — CHANGES_REQUESTED** [review]
please fill out the changelog correctly, you aren't adding an image, you're likely fixing a bug

**Comment by wonderinghost** [comment]
> please fill out the changelog correctly, you aren't adding an image, you're likely fixing a bug

Wasn't really a bug for the cuffs. They just left it like that. 

Also what would the changelog technically be. Before this the alien cuffs didn't have a sprite and had been like that for who knows how long.

File: code/game/objects/items/handcuffs.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	throw_speed = 3
 	throw_range = 5
 	materials = list(/datum/material/iron=500)
-	breakouttime = 600 //Deciseconds = 60s = 1 minute
+	breakouttime = 500 //Deciseconds = 50s
```
Comment (Moltijoe) [inline code/game/objects/items/handcuffs.dm]: just use time defines

File: code/game/objects/items/handcuffs.dm [hunk 2]
```diff
@@ -111,6 +111,10 @@
 		qdel(src)
 	return
 
+/obj/item/restraints/handcuffs/energy/used/swarmer //energy cuffs are in abductor, why would you do this
+	breakouttime= 200 // you already get teleported across the map
```
Comment (Moltijoe) [inline code/game/objects/items/handcuffs.dm]: here too

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 3]
```diff
@@ -585,7 +585,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 /obj/item/restraints/handcuffs/energy
 	name = "hard-light energy field"
 	desc = "A hard-light field restraining the hands."
-	icon_state = "cuff" // Needs sprite
+	icon_state = "handcuffAlien" //likely sprite change, enabling 
 	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
 	breakouttime = 450
```
Comment (Moltijoe) [inline code/modules/antagonists/abductor/equipment/abduction_gear.dm]: this will make cult handcuffs also use the new sprite

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm (hunk 3 above)
Comment (wonderinghost) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: well isnt it a spell?

Comment (Moltijoe) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: yes, but it spawns handcuffs

Comment (wonderinghost) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: ill hide it then

also i converted the handcuffs page over to seconds

File: code/game/objects/items/handcuffs.dm [hunk 4]
```diff
@@ -350,7 +354,7 @@
 	breakouttime = 30
 	item_flags = DROPDEL
 	flags_1 = NONE
-	break_strength = 2
+	break_strength = 2 SECONDS
```
Comment (Moltijoe) [inline code/game/objects/items/handcuffs.dm]: break_strength is not breakouttime

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/game/objects/items/handcuffs.dm (hunk 4 above)
Comment (wonderinghost) [reply code/game/objects/items/handcuffs.dm]: crap sorry

File: code/game/objects/items/handcuffs.dm [hunk 5]
```diff
@@ -377,7 +381,7 @@
 	righthand_file = 'icons/mob/inhands/weapons/thrown_righthand.dmi'
 	breakouttime = 35//easy to apply, easy to break out of
 	gender = NEUTER
-	break_strength = 3
+	break_strength = 3 SECONDS
```
Comment (Moltijoe) [inline code/game/objects/items/handcuffs.dm]: here too

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Comment by Therandomhoboo** [comment]
Swarmers kinda need the long cuff time because by the time they are discovered it is pretty much a countdown till the swarmers get eliminated and as for normal cuffs, there wasn't any problem nor enough kick-off to warrant a reduction in cuffs. At least I've not seen a ton of people kick off about it. 

Other then that its all gravy. Just don't agree with the reduction times in cuffing. Considering iirc that you can still be located if you have your suit sensors on and you can call out your position via comms if it aint down.

**Comment by Moltijoe** [comment]
Balance Council:
>No need to touch regular handcuffs, revert those

**Comment by wonderinghost** [comment]
> Balance Council:
> 
> > No need to touch regular handcuffs, revert those

should be a minute now

**Review by JamieD1 — APPROVED** [review]

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**